prov/efa/src/rxr/rxr_pkt_type_base.c

```diff
@@ -219,16 +219,62 @@ size_t rxr_pkt_data_size(struct rxr_pkt_entry *pkt_entry)
 		       pkt_type == RXR_DC_LONGCTS_MSGRTM_PKT ||
 		       pkt_type == RXR_DC_LONGCTS_TAGRTM_PKT ||
 		       pkt_type == RXR_DC_EAGER_RTW_PKT ||
-		       pkt_type == RXR_DC_LONGCTS_RTW_PKT);
+		       pkt_type == RXR_DC_LONGCTS_RTW_PKT ||
+		       pkt_type == RXR_RUNTREAD_MSGRTM_PKT ||
+		       pkt_type == RXR_RUNTREAD_TAGRTM_PKT);
 
-		return pkt_entry->pkt_size - rxr_pkt_req_hdr_size(pkt_entry);
+		return rxr_pkt_req_data_size(pkt_entry);
 	}
 
 	/* other packet type does not contain data, thus return 0
 	 */
 	return 0;
 }
 
+int rxr_ep_flush_queued_blocking_copy_to_hmem(struct rxr_ep *ep)
```

Comment:
could u add docstring?

Reply:
added in new revision.

Reply:
how much performance improvement we gain from the batch copy, compared to copying each time?